docs/build/guides/basics/send-and-receive-payments.mdx

```diff
@@ -7,90 +7,251 @@ description: Learn to send payments and watch for received payments on the Stell
-      // optional and does not affect how Stellar treats the transaction.
-      .addMemo(StellarSdk.Memo.text("Test Transaction"))
-      // Wait a maximum of three minutes for the transaction
-      .setTimeout(180)
-      .build();
-    // Sign the transaction to prove you are actually the person sending it.
-    transaction.sign(sourceKeys);
-    // And finally, send it off to Stellar!
-    return server.submitTransaction(transaction);
-  })
-  .then(function (result) {
-    console.log("Success! Results:", result);
+try {
+  await rpc.getAccount(destinationId);
+} catch (error) {
+  console.error("Error checking destination account:", error);
+  throw error;
+}
+
+// Now we also load the source account to build the transaction.
+let sourceAccount: StellarSdk.Account;
+try {
+  sourceAccount = await rpc.getAccount(sourceKeys.publicKey());
+} catch (error) {
+  console.error("Error checking source account:", error);
+  throw error;
+}
+
+// The next step is to parametrize and build the transaction object
+let transaction: StellarSdk.Transaction;
+try {
+  // Using the source account we just loaded we begin to assemble the transaction.
+  // We set the fee to the base fee, which is 100 stroops (0.00001 XLM).
+  // We also set the network passphrase to TESTNET.
+  transaction = new StellarSdk.TransactionBuilder(sourceAccount, {
+    fee: StellarSdk.BASE_FEE,
+    networkPassphrase: StellarSdk.Networks.TESTNET,
   })
-  .catch(function (error) {
-    console.error("Something went wrong!", error);
-    // If the result is unknown (no response body, timeout etc.) we simply resubmit
-    // already built transaction:
-    // server.submitTransaction(transaction);
+    // We then add a payment operation to the transaction oject.
+    // This operation will send 10 XLM to the destination account.
+    // Obs.: Not specifying a explicit source account here means that the
+    // operation will use the source account of the whole transaction, which we specified above.
+    .addOperation(
+      StellarSdk.Operation.payment({
+        destination: destinationId,
+        asset: StellarSdk.Asset.native(),
+        amount: "10",
+      })
+    )
+    // We include an optional memo which oftentimes is used to identify the transaction
+    // when working with pooled accounts or to facilitate reconciliation.
+    .addMemo(StellarSdk.Memo.id("1234567890"))
+    // Finally, we set a timeout for the transaction.
+    // This means that the transaction will not be valid anymore after 180 seconds.
+    .setTimeout(180)
+    .build();
+} catch (error) {
+  console.error("Error building transaction:", error);
+  throw error;
+}
+
+try {
+  // We sign the transaction with the source account's secret key.
+  transaction.sign(sourceKeys);
+} catch (error) {
+  console.error("Error signing transaction:", error);
+  throw error;
+}
+
+let sendTransactionResponse: StellarSdk.rpc.Api.SendTransactionResponse;
+try {
```

Comment:
Will `sendTransaction` throw an error if the status is `ERROR`? Or will it just return the JSON response normally? I don't think we need a `try/catch` unless it throws the error rather than returning it.